app/models/annotation/AnnotationService.scala

```diff
@@ -139,17 +139,18 @@ class AnnotationService @Inject()(
       dataSet: DataSet,
       dataSource: DataSource,
       tracingType: TracingType.Value,
-      withFallback: Boolean,
+      fallbackLayerNameOpt: Option[String],
       resolutionRestrictions: ResolutionRestrictions,
       organizationName: String,
       oldTracingId: Option[String] = None)(implicit ctx: DBAccessContext): Fox[(Option[String], Option[String])] = {
     def getFallbackLayer: Option[SegmentationLayer] =
-      if (withFallback) {
-        dataSource.dataLayers.flatMap {
+      // todo: probably should fail if the fallbackLayerName cannot be found
+      fallbackLayerNameOpt.map(fallbackLayerName =>
```

Comment:
@fm3 Can you have a look at my scala changes on this branch? You can also check out the branch and run the corresponding code by using the advanced "create explorational" modal and then picking a segmentation layer (one layer should be enough).

I think, there are at least 2 todos:
- ensuring that an error is thrown if a fallback layer name is passed which does not exist
- for the "make hybrid" route, the `fallbackLayerName` should be accepted as a GET parameter, too.

The bug which I mentioned earlier (that the fallback layer was ignored), was front-end only and is fixed now. So, no debugging necessary, I hope :)